### DISCUSSION TITLE
Firefox Extension (Not updated)

### DISCUSSION THREAD

**Discussion by jkowalleck** [AtomMaterialUI/a-file-icon-web#275] [Q&A]
Why is the Firefox Extension not updated anymore?

**Comment by Nerixyz** [comment]
Shameless ad: I [have a fork](https://github.com/Nerixyz/a-file-icon-web) where I publish a signed extension ([releases](https://github.com/Nerixyz/a-file-icon-web/releases) - not published on addons.mozilla.org). I didn't get around to do it through GitHub actions yet - that would be my goal. Furthermore, the `iconGenerator` submodule is updated on my fork.

See also: https://github.com/mallowigi/a-file-icon-web/pull/199#issuecomment-1312544500

**Comment by mallowigi** [comment]
Multiple reasons: the first one is that I don't have the time to maintain these plugins at the moment. But even when I do, the release process for Firefox is quite painful compared to Chrome so I kind of gave up. @Nerixyz if you want to become a maintainer please contact me thanks!
